File: Software Development Module/Revision/BMICalculator/BMICalculator.java
/*
BMICalculator.java
Billy Cussen
30/07/2020
*/

public class BMICalculator{

    //Data Members
    private double weight;
    private double height;
    private String BMI;

    //Constructor
    public BMICalculator(double weight, double height){
        this.weight = weight;
        this.height = height;
        this.BMI = new String();
    }

    //Compute and Getter
    public String getBMI(){
        this.BMI = "Your BMI is: "+((double)Math.round(this.weight/Math.pow(this.height,2)));
        return this.BMI;
    }
}

File: Python Course/Software Development Module - in Python/Revision/BMICalculator/BMICalculator.py
"""
BMICalculator.py
Billy Cussen
21/10/2020
"""

class myBMICalculator:
    def __init__(self, weight, height):
        self.weight = weight
        self.height = height

    def calculateBMI(self):
        return "Your BMI is: "+str(round(self.weight/pow(self.height,2),2))